tests/test_lib_tokens_push.py

```diff
@@ -102,9 +102,9 @@ def test_01_create_token(self):
                       "pubkey": self.smartphone_public_key_pem})
         self.assertEqual(token.get_tokeninfo("firebase_token"), "firebasetoken")
         self.assertEqual(token.get_tokeninfo("public_key_smartphone"), self.smartphone_public_key_pem)
-        self.assertTrue(token.get_tokeninfo("public_key_server").startswith("-----BEGIN RSA PUBLIC KEY-----\nMIICC"),
+        self.assertTrue(token.get_tokeninfo("public_key_server").startswith(b"-----BEGIN RSA PUBLIC KEY-----\nMIICC"),
```

Comment:
I think we mustn't use bytestrings here, because tokeninfo values are always unicode strings. If we use bytestrings, the test fails under Python 3.

Reply:
I can check this. But this whole branch will currently not run under python3, since it is based on a rather old version of master.

Reply:
done in 9e0e477d